Makefile

```diff
@@ -7,11 +7,13 @@ PROJECTNAME=$(shell basename "$(PWD)")
 # Make is verbose in Linux. Make it silent.
 MAKEFLAGS += --silent
 
+GOARCH ?= $(shell go env GOARCH) # detect automatically the underlying arch
+
 compile: get build
 
 build:
 	@echo "  >  Building binaries..."
-	@CGO_ENABLED=0 go build -o ${PROJECTNAME} ./cmd/...
+	@CGO_ENABLED=0 GOARCH=$(GOARCH) go build -o ${PROJECTNAME} ./cmd/...
```

Comment:
Mark also proposed to add `GOOS` https://github.com/opiproject/opi-evpn-bridge/pull/377#discussion_r1561594650

Reply:
done